src/Goerli-DssSpell.t.sol

```diff
@@ -406,17 +406,17 @@ contract DssSpellTest is DSTest, DSMath {
             calc_cut:     9900
```

Comment:
We can remove L45/58 once the `dss-interfaces` is upgraded.
Also, we'd need to replace `DssVestLike` with `VestAbstract` in tests as well, to make them working with the right interface.